.eslintrc.json

```diff
@@ -23,5 +23,12 @@
     "prefer-const": "error",
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": "error"
-  }
+  },
+  "overrides": [
```

Comment:
This was necessary because otherwise ESlint would fail when trying to parse the `.js` files in the `sample-designs` directory.  Just saying here to use the default JS parser (`espree`) when evaluating JS files.